stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/CSS.java

```diff
@@ -99,8 +99,13 @@ public JsonRpcResult getMatchedStylesForNode(JsonRpcPeer peer, JSONObject params
     final GetMatchedStylesForNodeResult result = new GetMatchedStylesForNodeResult();
 
     final RuleMatch match = new RuleMatch();
+    final RuleMatch accessibilityMatch = getAccessibilityRuleMatch();
 
-    result.matchedCSSRules = ListUtil.newImmutableList(match);
+    List<RuleMatch> matchedCSSRules = new ArrayList<>();
+    matchedCSSRules.add(match);
+    matchedCSSRules.add(accessibilityMatch);
+
+    result.matchedCSSRules = ListUtil.copyToImmutableList(matchedCSSRules);
```

Comment:
in this case, since you're always populating the `List` with 2 elements, just use `ListUtil.newImmutableList` directly. You'll need to add a 2 parameter version that instantiates `TwoItemImmutableList`.